UI/EncyclopediaDetailPanel.cpp

```diff
@@ -2439,7 +2439,12 @@ namespace {
             std::string species_name = *it;
 
             std::string species_name_column1 = str(FlexibleFormat(UserString("ENC_SPECIES_PLANET_TYPE_SUITABILITY_COLUMN1")) % UserString(species_name)); 
-            max_species_name_column1_width = std::max(font->TextExtent(species_name_column1).x, max_species_name_column1_width);
+            std::vector<GG::Font::LineData> lines;
+            GG::Flags<GG::TextFormat> format = GG::FORMAT_NONE;
+            std::vector<boost::shared_ptr<GG::Font::TextElement> > text_elements =
+                font->ComputeTextElements(species_name_column1, format);
+            GG::Pt extent = font->DetermineLines(species_name_column1, format, GG::X(1 << 15), text_elements, lines);
```

Comment:
`GG::X(1 << 15)` is magic². What does it do?  An arbitrary huge value? A constant and some documentation would help.  Also remember there are other uses of this pattern when updating this PR.